### ISSUE TITLE
Warnings and an error when building nunaliit-es6

### ISSUE THREAD

**Issue by ahayes** [GCRC/nunaliit#998]
I'm not sure exactly when these lines started to appear in the build output. Can something be done to suppress the warnings and resolve the error? The audit bits seem to point to a need to update some dependencies. I could not sort out how to run npm audit since it is being run within an environment created by the maven plugin. I imagine some temporary magic in `nunaliit2-js/src/main/nunaliit-es6/package.json` might get the build to spit out the audit details.

```
[INFO] --- frontend-maven-plugin:1.4:npm (npm-install-dependency) @ nunaliit2-js ---
[INFO] Running 'npm install' in /home/ahayes/nunaliit/nunaliit2-js/src/main/nunaliit-es6
[WARNING] npm WARN n2es6@1.0.0 No repository field.
[WARNING] npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/fsevents):
[WARNING] npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
[WARNING] npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/watchpack/node_modules/fsevents):
[WARNING] npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
[ERROR] 
[INFO] audited 1601 packages in 29.176s
[INFO] found 135 vulnerabilities (6 low, 78 moderate, 46 high, 5 critical)
[INFO]   run `npm audit fix` to fix them, or `npm audit` for details
```

**Comment by ahayes** [comment]
Note that the build ultimately succeeds but given that the error refers to vulnerabilities, it would be good to investigate.

**Comment by billingb** [comment]
Fixed by #1067
